electron/src/holochain.ts

```diff
@@ -65,14 +65,14 @@ const COMMUNITY_PROXY_URL =
 const devOptions: ElectronHolochainOptions = {
   happPath: profilesHappPath, // preload
   datastorePath: process.env.ACORN_TEST_USER_2
-    ? '../user2-data/databases'
-    : path.join(__dirname, '../../user-data/databases'),
+    ? path.join(__dirname, `../../user2-data/databases-${INTEGRITY_VERSION_NUMBER}`)
+    : path.join(__dirname, `../../user-data/databases-${INTEGRITY_VERSION_NUMBER}`),
```

Comment:
can you push a step further on this refactor and consolidate this with the other references like we talked about? add another file called `paths` or some such like name and export constants from it that this file and the other import from.